src/pages/dashboard/components/menu/menu.style.tsx

```diff
@@ -3,5 +3,6 @@ import styled from '@emotion/styled'
 export const MenuStyled = styled('nav')`
   padding: 10px;
 
-  background: #cccccc;
+  background: #bd1414;
+  width: 4rem;
```

Comment:
rem unit is more for font-sizes,  so use px. but i think its not needed here at all